phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/standard/main.less

```diff
@@ -946,3 +946,24 @@
 .aboutButton {
   -fx-background-image: url('/org/phoenicis/javafx/themes/standard/icons/mainwindow/settings/about.png');
 }
+
+/*********************** filter-icons **********************/
+.testingIcon {
+  -fx-background-image: url('/org/phoenicis/javafx/filter/testing.png');
```

Comment:
The image files are in the wrong folder I think. Ideally you should place them somewhere in `/org/phoenicis/javafx/themes/standard/icons` or `/org/phoenicis/javafx/themes/standard/`